transfermethodui/src/androidTest/java/com/hyperwallet/android/ui/transfermethod/SelectTransferMethodTest.java

```diff
@@ -446,4 +482,118 @@ public void testSelectTransferMethod_verifyDefaultsToUSWhenUserProfileDoesNotHav
+
+        onView(withId(R.id.select_transfer_method_currency_container)).perform(click());
+        UiDevice mDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation());
+        mDevice.pressBack();
+
+        verify(mHyperwalletInsight, atMost(1)).trackImpression(any(Context.class), anyString(), anyString(),
+                ArgumentMatchers.<String, String>anyMap());
+    }
+
+    @Test
+    public void testSelectTransferMethod_verifyClickEventCreatedWhenUserMakesWireTransferMethodSelection() {
+        mMockWebServer.mockResponse().withHttpResponseCode(HTTP_OK).withBody(sResourceManager
+                .getResourceContent("user_response.json")).mock();
+        mMockWebServer.mockResponse().withHttpResponseCode(HTTP_OK).withBody(sResourceManager
+                .getResourceContent("successful_tmc_keys_response.json")).mock();
+        mMockWebServer.mockResponse().withHttpResponseCode(HTTP_OK).withBody(sResourceManager
+                .getResourceContent("successful_tmc_fields_bank_card_response.json")).mock();
+
+        mActivityTestRule.launchActivity(null);
+
+        onView(withId(R.id.select_transfer_method_types_list))
+                .perform(RecyclerViewActions.actionOnItem(withChild(withText(R.string.wire_account)), click()));
+        onView(allOf(instanceOf(TextView.class), withParent(withId(R.id.toolbar)))).check(
+                matches(withText("Wire Transfer")));
+
+        verify(mHyperwalletInsight, atLeastOnce()).trackClick(any(Context.class), anyString(), anyString(), anyString(),
+                ArgumentMatchers.<String, String>anyMap());
+    }
+
+    @Test
+    public void testSelectTransferMethod_verifyClickEventCreatedWhenUserMakesPayPalTransferMethodSelection() {
+        mMockWebServer.mockResponse().withHttpResponseCode(HTTP_OK).withBody(sResourceManager
+                .getResourceContent("user_response.json")).mock();
+        mMockWebServer.mockResponse().withHttpResponseCode(HTTP_OK).withBody(sResourceManager
+                .getResourceContent("successful_tmc_keys_response.json")).mock();
+        mMockWebServer.mockResponse().withHttpResponseCode(HTTP_OK).withBody(sResourceManager
+                .getResourceContent("successful_tmc_fields_bank_card_response.json")).mock();
+
+        mActivityTestRule.launchActivity(null);
+
+        onView(withId(R.id.select_transfer_method_types_list))
+                .perform(RecyclerViewActions.actionOnItem(withChild(withText("PayPal Account")), click()));
+        onView(allOf(instanceOf(TextView.class), withParent(withId(R.id.toolbar)))).check(
+                matches(withText("PayPal Account")));
+
+        verify(mHyperwalletInsight, atLeastOnce()).trackClick(any(Context.class), anyString(), anyString(), anyString(),
+                ArgumentMatchers.<String, String>anyMap());
+    }
+
+    @Test
+    public void testSelectTransferMethod_verifyClickEventCreatedWhenUserMakesLongClickOnTransferMethod() {
```

Comment:
You can remove this test, we don't need to automate this.

Reply:
Removed.